library/network/src/lib/y2firewall/firewalld.rb

```diff
@@ -92,6 +121,12 @@ def modified?
 
     # Apply the changes to the modified zones and sets the logging option
     def write
+      write_only
+      reload
```

Comment:
i know that ```write_only``` returns always true ... but may be that ```write_only && reload``` would be safer for the future